### PR TITLE
Support icon type

### PR DESCRIPTION
**Pull request by huangshihai**

### CODE REVIEW

**Comment by xKiian** [comment]
im open to merge this, but i gotta ask, how did you get the geetest_v4_icon.onnx? would you be open to share the training process of the model?

**Comment by xKiian** [comment]
also i can not seem to find the "ddddocr" package. When i run `pip install ddddocr` i get this error:
```
DdddOcr.__init__() got an unexpected keyword argument 'det'
```

**Comment by huangshihai** [comment]
# Notes on ddddocr and geetest_v4_icon.onnx modeling

1. **ddddocr** From the open source project:  
   [ddddocr](https://github.com/sml2h3/ddddocr)

2. **geetest_v4_icon.onnx model** is trained by ddddocr's support training tool:  
   [dddd_trainer](https://github.com/sml2h3/dddd_trainer)  
   The specific training deployment process is as follows:

   ### Training steps
   - 2.1 Download the icon type CAPTCHA, use ddddocr's target recognition function to detect and crop the target image.
   - 2.2 Manually label the target image, and finally obtain 10,000+ labeled data.
   - 2.3 Use dddd_trainer to create a CNN project for training.

3. **The training dataset** has been uploaded to the web disk:  
   [images_set](https://wwrd.lanzoum.com/i2ajU2u32sqh)

Translated with DeepL.com (free version)
